climada/hazard/centroids/centr.py

```diff
@@ -821,13 +818,20 @@ def __deepcopy__(self, memo):
         return result
 
 
-def generate_nat_earth_centroids(res_as=360):
+def generate_nat_earth_centroids(res_as=360, path=None, dist_coast=False):
     """For reproducibility, this is the function that generates the centroids
-        files `NATEARTH_CENTROIDS_*AS`. These files are provided with CLIMADA
+        files in `NATEARTH_CENTROIDS`. These files are provided with CLIMADA
         so that this function should never be called!
 
     Parameters:
         res_as (int): Resolution of file in arc-seconds. Default: 360.
+        path (str, optional): If set, write resulting hdf5 file here instead of
+            the default location.
+        dist_coast (bool): Should the distance to coast be read from a NASA
+            dataset? (see util.coordinates.dist_to_coast_nasa)
```

Comment:
A parameter docstring shouldn't have the form of a question. Use instead: "If True, include distance to coast information in the output file. Default: False."